contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -277,21 +279,79 @@ contract WithdrawalQueue is AccessControlEnumerable, WithdrawalQueueBase {
         return _requestWithdrawalWstETH(_amountOfWstETH, _recipient);
     }
 
-    /// @notice Request withdrawal of the provided token amount in a batch
-    /// NB: Always reverts
-    function requestWithdrawalBatch(uint256[] calldata, address[] calldata)
+    struct WithdrawalRequestInput {
```

Comment:
nice idea :+1:

Reply:
Is there any drawbacks in reusing these structs in all the methods?

Reply:
Or we can even get rid of the singular methods at all. I like this idea more and more.

Reply:
@DiRaiks fyi